scalyr_agent/monitor_utils/k8s.py

```diff
@@ -1101,8 +1136,8 @@ def pod( self, namespace, name, current_time=None, query_options=None ):
         if local_state.k8s is None:
             return
 
-        return self._pods.lookup( local_state.k8s, namespace, name, kind='Pod', current_time=current_time,
-                                  query_options=query_options )
+        return self._pods_cache.lookup(local_state.k8s, namespace, name, kind='Pod', current_time=current_time,
```

Comment:
Rename `pods_cache` for clarity.  Another variable that is somewhat hard to keep straight (i.e a lot of `pods` floating around)